### PR TITLE
added rating schema

### PR DESCRIPTION
**Pull request by kailash-bitpack**

### CODE REVIEW

File: apps/server/prisma/schema.prisma [hunk 1]
```diff
@@ -67,6 +67,28 @@ model Views {
...
 
+model Ratings {
+  id            Int       @id @default(autoincrement())
+  installId     Int?
+  viewId        Int?
+  dappId        String    @db.VarChar
+  dappVersion   String?   @db.VarChar
+  userId        String?   @db.VarChar
+  userAddress   String?   @db.VarChar
+  device        String?   @db.VarChar
```
Comment (r4881t) [inline apps/server/prisma/schema.prisma:78]: Not needed.

File: apps/server/prisma/schema.prisma [hunk 2]
```diff
@@ -67,6 +67,28 @@ model Views {
   @@index([visitDate])
 }
 
+model Ratings {
+  id            Int       @id @default(autoincrement())
+  installId     Int?
```
Comment (r4881t) [inline apps/server/prisma/schema.prisma:72]: Why?

File: apps/server/prisma/schema.prisma [hunk 3]
```diff
@@ -67,6 +67,28 @@ model Views {
...
+  viewId        Int?
```
Comment (r4881t) [inline apps/server/prisma/schema.prisma:73]: Why?

Comment (kailash-bitpack) [reply apps/server/prisma/schema.prisma:73]: if dapp/PWA is viewed, then we can store as a reference, installId for dappInstallation, viewId for dapp visited id

Comment (r4881t) [reply apps/server/prisma/schema.prisma]: Why?

**Review by r4881t — APPROVED** [review]

**Pull request merged**